include/aspect/geometry_model/interface.h

```diff
@@ -130,6 +130,18 @@ namespace aspect
         double depth(const Point<dim> &position) const = 0;
 
         /**
+         * TODO
+         */
+        virtual
```

Comment:
in 2d the coordinates for the current spheroid geometries are (radius, longitude)